PrebidMobile/AdUnits/AdUnit.swift

```diff
@@ -305,22 +305,26 @@ public class AdUnit: NSObject, DispatcherDelegate {
     
     /// This method obtains the ext data keyword & value for adunit targeting.
     /// If the key already exists the value will be appended to the list. No duplicates will be added
+    @available(*, deprecated, message: "This method is deprecated. In the upcoming major release, the method will be removed.")
```

Comment:
We need to clearly indicate in the message which alternative method should be used in place of the deprecated one. This could be achieved either through a global-level method from the `Targeting` class or by using the `setImpORTBConfig`/`setGlobalORTBConfig` methods.